graphql/resolvers/Author.js

```diff
@@ -0,0 +1,93 @@
+import { Op } from 'sequelize';
+import { Author } from '../../db-schema/postgres/Author.js';
+import { AuthorDetail } from '../../db-schema/mongo/AuthorDetail.js';
+
+const batchAuthors = async (authorIds) => {
+    const authors = await Author.findAll({
+      where: {
+        id: {
+            [Op.in]: authorIds,
+        },
+      },
+    });
+    return authorIds.map(id => authors.find(author => author.id === id) || null);
```

Comment:
It can become slow for a large number of authors. Can we create an authorId -> Author mapping and then use that?

Reply:
Both are in the same table so it will take the same time with that approach. This logic was for parsing the object types. But I understand how it can be further optimized.